streams/src/test/java/org/apache/kafka/streams/integration/EosIntegrationTest.java

```diff
@@ -713,7 +690,7 @@ private void writeInputData(final List<KeyValue<Long, Long>> records) throws Exc
     }
 
     private List<KeyValue<Long, Long>> readResult(final int numberOfRecords,
-                                                  final String groupId) throws InterruptedException {
+                                                  final String groupId) throws Exception {
```

Comment:
Why we want to generalize the exception to throw?

Reply:
Because it's the error case -- nobody is supposed to catch any exception and the test fails anyway. This is the way I was taught to write unit tests with good code style.